scripts/download_and_archive.sh

```diff
@@ -1,103 +1,43 @@
 #!/usr/bin/env bash
 
-set -euxo pipefail
+# Fail on error
+set -o errexit
+# Disable wildcard character expansion
+set -o noglob
+# Disable undefined variable reference
+set -o nounset
```

Comment:
set -o pipefail

is also wanted, don't you think?

Reply:
1 minute